.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: python
+
+python:
+  #- 2.5 #Travis was unable to access Python<=2.5
+  - 2.6
```

Comment:
I think it should be sufficient to test with 2.7, because that is the version we use in newer XenServers?

Reply:
If I recall correctly v2.5 was what I saw on one of our test servers I orchestrated recently. I'll double check when I am next in. We may need to ensure we use the latest versions internally.

Reply:
You are correct , I misread my notes 2.7.5 is what we use on our test servers, I'll remove the test for 2.6.